pkg/backup/schema.go

```diff
@@ -84,9 +86,14 @@ func (ss *Schemas) BackupSchemas(
 	workerPool := utils.NewWorkerPool(concurrency, "Schemas")
 	errg, ectx := errgroup.WithContext(ctx)
 	startAll := time.Now()
+	op := metautil.AppendSchema
+	metaWriter.StartWriteMetasAsync(ctx, op)
 	for _, s := range ss.schemas {
 		schema := s
 		workerPool.ApplyOnErrorGroup(errg, func() error {
+			if utils.IsSysDB(schema.dbInfo.Name.L) {
+				schema.dbInfo.Name = utils.TemporaryDBName(schema.dbInfo.Name.O)
```

Comment:
Why to change the table schema in backup?

Reply:
we need to ensure `-f` can match sysDB. if we want to restore sysDB.